zeropilot3.5/include/system_manager/system_manager.hpp

```diff
@@ -6,17 +6,18 @@
 
 class SystemManager {
     public:
-    SystemManager(SBusIface *rc_driver, QueueIface<RCMotorControlMessage_t> *queue_driver);
+    SystemManager(ISBUSReceiver *rc_driver, IMessageQueue<RCMotorControlMessage_t> *queue_driver, int16_t invalid_threshold=50);
```

Comment:
make `invalid_threshold` a `uint32_t`

remove the default value on `invalid_threshold`. we want to force calculation of a value when we spawn the thread w.r.t. thread run rate so that we can say failsafe triggers after x seconds without rc connection